azurerm/internal/services/recoveryservices/resource_arm_recovery_services_vault.go

```diff
@@ -67,13 +67,25 @@ func resourceArmRecoveryServicesVault() *schema.Resource {
 				Optional: true,
 				Default:  true,
 			},
+
+			"storage_replication_type": {
+				Type:     schema.TypeString,
+				Optional: true,
+				Default:  "GRS",
+				ValidateFunc: validation.StringInSlice([]string{
+					"LocallyRedundant",
+					"GeoRedundant",
+				}, true),
+				DiffSuppressFunc: suppress.CaseDifference,
```

Comment:
could we make this case snesitive
```suggestion
				}, false),
```